lambdamud-client/src/App.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Route, withRouter } from 'react-router-dom';
+import Register from './components/Register';
+import { createGlobalStyle } from 'styled-components';
+import { connect } from 'react-redux';
+import * as actions from './actions';
+import Login from './components/Login';
+import Nav from './components/Nav';
+import Game from './components/Game';
+
+const GlobalStyle = createGlobalStyle`
+  @font-face{
+    font-family: 'radnika_next';
+    src: require(url('/static/radnikanext_medium-webfont.woff2'))
+    format('woff2');
+    font-weight: normal;
+    font-style: normal;
+  } 
+  html {
+    box-sizing: border-box;
+    font-size: 10px;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  body {
+    padding: 0;
+    margin: 0;
+    font-size: 1.5rem;
+    line-height: 2;
+    font-family: 'radnika_next';
+  }
+  a {
+    text-decoration: none;
+    color: black;
+  }
+`;
+
```

Comment:
I usually put my css into a separate file and import it. Does it work better to put it in the App.js file like this?

Reply:
I'm using styled components, so it's a little bit different, but that would be perfectly fine too!